spyder/plugins/toolbar/container.py

```diff
@@ -153,6 +161,12 @@ def create_application_toolbar(self, toolbar_id, title):
             toolbar, toolbar_id, self.PLUGIN_NAME, self.CONTEXT_NAME)
         self._APPLICATION_TOOLBARS[toolbar_id] = toolbar
 
+        if toolbar_id in self._ITEMS_QUEUE:
+            pending_items = self._ITEMS_QUEUE.pop(toolbar_id)
+            for item, section, before, before_section in pending_items:
+                toolbar.add_item(item, section=section, before=before,
+                                 before_section=before_section)

```

Comment:
Please move this code to a method because it's repeated in two different places.